interpreter/cling/test/lit.cfg

```diff
@@ -334,8 +334,16 @@ config.substitutions.append(('%testexecdir', config.test_exec_root))
 config.substitutions.append(('%llvmsrcdir', config.llvm_src_root))
 config.available_features.add('vanilla-cling')
 
-if lit.util.which('libcudart.so', config.environment.get('LD_LIBRARY_PATH','')) is not None:
+libcudart_path = lit.util.which('libcudart.so', config.environment.get('LD_LIBRARY_PATH',''))
+if libcudart_path is not None:
   config.available_features.add('cuda-runtime')
+  # set the CUDA SDK root path
+  # necessary if CUDA is not installed under /usr/local/
+  config.substitutions.append(('%cudapath', libcudart_path[0:-len('/lib64/libcudart.so')]))
+  # limit the number of usable GPUs in a system
+  # https://developer.nvidia.com/blog/cuda-pro-tip-control-gpu-visibility-cuda_visible_devices/
+  if 'CUDA_VISIBLE_DEVICES' in os.environ:
+      config.environment['CUDA_VISIBLE_DEVICES'] = os.environ['CUDA_VISIBLE_DEVICES']
```

Comment:
Where is `config.environment['CUDA_VISIBLE_DEVICES']` used?

Reply:
Inside the tests. If you do not manually select the GPU in your source code, CUDA runtime will choose the GPU with id 0 by default. In our workstation system, where we have two completely different GPUs, this can cause problems. With `CUDA_VISIBLE_DEVICES` you can limit the GPUs that the application can see and thus indirectly select the correct GPU.